src/core/Akka.Remote/EndpointRegistry.cs

```diff
@@ -144,7 +146,8 @@ public EndpointManager.EndpointPolicy WritableEndpointWithPolicyFor(Address addr
 
         public bool HasWriteableEndpointFor(Address address)
         {
-            return WritableEndpointWithPolicyFor(address) != null;
+            var policy = WritableEndpointWithPolicyFor(address);
+            return policy is EndpointManager.Pass;
```

Comment:
This was a bad bug - previously we're return true, that a node is writeable as long as it had ANY policy. Not true - needs to be `Pass` in order to be writeable.